clang/lib/DPCT/Diagnostics.inc

```diff
@@ -211,3 +211,5 @@ DEF_WARNING(PRIMITIVE_WORKSPACE, 1097, "The function \"%0\" may require the work
 DEF_COMMENT(PRIMITIVE_WORKSPACE, 1097, "The function \"{0}\" may require the workspace used to save intermediate results from function \"{1}\". By default, a workspace from engine_ext is selected according to the source data pointer, but this may be incorrect and cause a workspace data race. You may need to rewrite this code.")
 DEF_WARNING(MATH_EMULATION_EXPRESSION, 1098, "The %1 expression is used instead of the %0 call. These two expressions do not provide the exact same functionality. Check the generated code for potential precision and/or performance issues.")
 DEF_COMMENT(MATH_EMULATION_EXPRESSION, 1098, "The {1} expression is used instead of the {0} call. These two expressions do not provide the exact same functionality. Check the generated code for potential precision and/or performance issues.")
+DEF_WARNING(MACRO_EXPR_REPLACED, 1099, "'%0' expression was replaced with a value. Modify the code to use original expression, provided in comments, if it is correct.")
```

Comment:
1099 message ID is already reserved by other diagnostics (FFT-related). Please, update the ID.

Reply:
updated, and because the ID must be consecutive, so I need move the syclct IDs to here.